python-frontend/src/main/java/org/sonar/python/semantic/ClassSymbolImpl.java

```diff
@@ -46,15 +53,34 @@
   private Map<String, Symbol> membersByName = null;
   private boolean hasAlreadyReadSuperClasses = false;
   private boolean hasAlreadyReadMembers = false;
+  private final LocationInFile classDefinitionLocation;
+
+  public ClassSymbolImpl(ClassDef classDef, @Nullable String fullyQualifiedName, PythonFile pythonFile) {
```

Comment:
I think we should use this constructor also here: https://github.com/SonarSource/sonar-python/blob/d77026425a97a4dd3fda0eda3d834bb6f13c682b/python-frontend/src/main/java/org/sonar/python/semantic/SymbolTableBuilder.java#L191